scripts/startLocalCluster.sh

```diff
@@ -157,7 +157,8 @@ for aid in `seq 0 $(( $NRAGENTS - 1 ))`; do
           --server.endpoint $TRANSPORT://$ENDPOINT:$PORT \
           --server.statistics false \
           --log.file cluster/$PORT.log \
-          --log.force-direct true \
+          --log.force-direct false \
+          --log.use-microtime true \
```

Comment:
is this a temp change or permanent change?  looks like debugging change to me.

Reply:
I have not yet made up my mind. I think the force-direct false is at least my intention to be permanent. The use-microtime is often necessary for accurate timings, but is rather annoying because one loses the wallclock time comparison. What I really wished was that use-microtime would show a proper timestamp down to seconds and then just add microseconds accuracy. Maybe I will implement this.

Reply:
I will leave force-direct but revert the microtime. Jan has a PR to implement a time format, there we should change this to something sensible.